css/style.css

```
@@ -0,0 +1,170 @@
+/* Variables */
+
+:root{
+  --basic-black: #000000;
+  --basic-white: #ffffff;
+  --basic-gray: #333333;
+  --basic-light-gray: #F2F2F2;
+  --basic-brown: #766357;
+  --basic-blue: #81B3D2;
+  --background-gray: #F2F2F2;
+  --hover-brown: #604E43;
+  --active-brown: #503E33;
+  --hover-blue: #669EC0;
+  --active-blue: #5496BD;
+}
+
+/*Global*/
+
+body{
+  margin: 0;
+  padding: 0;
+  font-family: "PT Sans Narrow", Arial, sans-serif;
+  font-size: 14px;
+  line-height: 21px;
+  font-weight: 400;
+  color: var(--basic-gray);
+  text-transform: uppercase;
+  background-color: var(--basic-light-gray);
+}
+
+.wrap{
+  width: 100%;
+  max-width: 1200px;
+  margin: auto;
+  background-color: var(--basic-white);
+}
+
+a{
+  text-decoration: none;
+}
+
+img{
+  max-width: 100%;
+  height: auto;
+}
+
+.button{
+  background: var(--basic-brown);
+  text-transform: uppercase;
+  color: var(--basic-white);
+  border: none;
+}
+
+.button:hover,
+.button:focus{
+  background: var(--hover-brown);
+  border: none;
+}
+
+.button:active{
+  background: var(--active-brown);
+  border: none;
+}
+
+/*Navigation*/
+
+.main-navigation{
+  line-height: 26px;
+  font-weight: 700;
+}
+
+.site-navigation{
+  list-style: none;
+}
+
+.site-navigation a{
+  color: var(--basic-black);
+}
+
+/*Page title*/
+
+.main-title{
+  background: url("../img/background.jpg") no-repeat center;
+}
+
+.main-title::after{
+  content: url("../img/intro-mask.svg");
+}
```

Здесь лучше запиши так

```suggestion
.main-title::after {
    content: '';
    display: block;
    height: 57px;
    background-image: url(../img/intro-mask.svg);
}
```
Картинка встанет ровно